src/common/ui-components/MenuIcon/MenuIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import styles from './MenuIcon.module.scss';
+
+const MenuIcon: React.FC<any> = ({ isDark, isOpen }) => {
+  return (
+    <div
+      className={classNames(
+        isOpen
+          ? styles.navIcon3Open
+          : isDark
+          ? styles.navIcon3Dark
+          : styles.navIcon3
+      )}
+    >
+      <span></span>
```

Comment:
Not the biggest fan of this, seems hacky. But the end result looks good.

Reply:
of what exactly?